app/src/main/java/com/keylesspalace/tusky/fragment/SFragment.java

```diff
@@ -66,6 +70,7 @@
     protected abstract void onReblog(final boolean reblog, final int position);
 
     private BottomSheetActivity bottomSheetActivity;
+    private String openAsTitle;
```

Comment:
Its weird that this is a member variable and not a local one. Can you move it somehow, maybe set the title to `openAsItem` directly?

Reply:
Good point - the compiler was hating because I was passing it to the lambda, but I can just get the title from `openAsItem` at that point